method/GraphST/method_GraphST.py

```diff
@@ -177,6 +191,12 @@ def get_anndata(args):
 adata = get_anndata(args)
 adata.var_names_make_unique()
 
+# https://github.com/JinmiaoChenLab/GraphST/blob/main/GraphST/preprocess.py
+sc.pp.highly_variable_genes(adata, flavor="seurat_v3", n_top_genes=n_genes)
```

Comment:
Looking all good! I assume for Visium and StereoSeq all their gene numbers will be larger than 3k right?

Reply:
If the number of genes is greater than the specified `n_top_genes`, `sc.pp.highly_variable_genes` will reset `n_top_genes to adata.n_vars`.